agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -1582,6 +1582,19 @@ bool slave_thread::handle_cmdu_platform_manager_message(
                 send_cmdu_to_controller(cmdu_tx);
             }
 
+            // notify monitor
+            auto monitor_notification = message_com::create_vs_message<
+                beerocks_message::cACTION_MONITOR_CLIENT_NEW_IP_ADDRESS_NOTIFICATION>(cmdu_tx);
+            if (!monitor_notification) {
+                LOG(ERROR) << "Failed building message!";
+                return false;
+            }
+
+            monitor_notification->mac()  = notification->mac();
+            monitor_notification->ipv4() = notification->ipv4();
+
+            message_com::send_cmdu(monitor_socket, cmdu_tx);
+
```

Comment:
Indeed. So instead of start_monitoring, it should be a message that sets a station's IP address and bridge MAC address.

Reply:
Just to be clear: the ACTION_DHCP_LEASE_ADDED_NOTIFICATION will only "happen" on the agent of the gateway, because that has the DHCP server. That agent sends it to the controller, then the controller updates the IP address in the database and sends it to all agents (not only the local one).

Reply:
The controller sends it to the agent that the client is connected to.
But maybe the Agent of the GW could send it in multicast. That should work.